src/components/StatusIndicator/StatusStep/StatusStep.js

```diff
@@ -80,7 +82,7 @@ const getStatusIcon = (status, description) => {
  */
 class StatusStep extends Component {
   static propTypes = {
-    /** @type {string} A label for a step. */
+    /** @type {string} Used to create a custom key for a step. */
     label: string.isRequired,
```

Comment:
This prop is used to apply extra characters to a `key` 🤔 I think it needs to be re-evaluated... but for now I can update the prop desc.